src/Models/Business.js

```diff
@@ -0,0 +1,109 @@
+class Business {
+	constructor(name, owners, description, location, hours, category, products, reviews) {
+		this.name = name;
+		this.owners = pwners;
+		this.description = description;
+		this.location = location;
+		this.hours = hours;
+		this.category = category;
+		this.products = products;
+		this.reviews = reviews;
+	}
+
+	getName() {
+		return this.name;
+	}
+
+	getOwners() {
+		return this.owners;
+	}
+
+	getDescription() {
+		return this.description;
+	}
+
+	getLocation() {
+		return this.location;
+	}
+
+	getHours() {
+		return this.hours;
+	}
+
+	getCategory() {
+		return this.category;
+	}
+
+	getProducts() {
+		return this.products;
+	}
+
+	getReviews() {
+		return this.reviews;
+	}
+
+	setName(new_name) {
+		this.name = new_name;
+	}
+
+	setOwners(new_owners) {
+		this.owners = new_owners;
+	}
+
+	addOwners(new_owners) {
+		this.owners.push(new_owners);
+	}
+
+	deleteOwner(old_owner) {
+		old_index = this.owners.indexOf(old_owner);
+		if (old_index > -1) {
+    		this.owners.splice(old_index, 1);
+        }
+	}
```

Comment:
all what this function will do will be to provide an id to the graphql mutation requestion to delete it.